solr/core/src/java/org/apache/calcite/config/CalciteConnectionProperty.java

```diff
@@ -0,0 +1,182 @@
+/*
```

Comment:
This whole files needs to be removed when Calcite upgrades to Avatica 1.9 and releases 1.11 most likely